app/utils/kcidb.py

```diff
@@ -144,8 +158,13 @@ def push_build(build_id, first, bq_options, db_options={}, db=None):
             build[models.FILE_SERVER_RESOURCE_KEY],
             build[models.KERNEL_CONFIG_KEY],
         ]),
+        'output_files': output_files,
         'misc': {
-            'defconfig': build[models.DEFCONFIG_FULL_KEY],
+            'kernel_image_size': build[models.KERNEL_IMAGE_SIZE_KEY],
+            'vmlinux_bss_size': build[models.VMLINUX_BSS_SIZE_KEY],
+            'vmlinux_data_size': build[models.VMLINUX_DATA_SIZE_KEY],
+            'build_platform': build[models.BUILD_PLATFORM_KEY],
+            'build_time': build[models.BUILD_TIME_KEY],
```

Comment:
There's already a generic `duration` field for this: https://github.com/kernelci/kcidb/blob/master/kcidb/db/schema.py#L124
which is in fact populated a bit earlier in this function.

So while I think the other parts are less trivial to add to a common schema, I think `build_time` should be dropped from `misc`.

Reply:
Agreed, forgot about the existing `duration`.  Will remove.